src/com/theironyard/novauc/Main.java

```diff
@@ -1,8 +1,93 @@
 package com.theironyard.novauc;
 
+import java.util.Scanner;
+
+import static java.lang.System.*;
+
+
 public class Main {
+    public static double availableBal = 2357;
+    public static double totalBal = 2357;
+    public static Scanner scanner = new Scanner(in);
+    private static Scanner input;
+
+
+    public static void main(String[] args) throws Exception {
+        // write your code here
+
+
+        boolean run = true;
+        while (run) {
+
+            System.out.println("Welcome, Customer.");
+            out.println(" What is your name?");
+
+            Scanner scanner = new Scanner(in);
+
+
+            String name = scanner.nextLine();
 
-    public static void main(String[] args) {
-	// write your code here
+            if (name.isEmpty()) {
+                throw new Exception("Enter Name");
+            } else {
+                System.out.println("Welcome ," + name);
+
+            }
+            System.out.println("What would you like to do? 1. Display Balance 2. Withdrawal 3. Cancel");
+            String transaction = scanner.nextLine();
+            // make a decision based on what they input
+            // if statement to make the decision
+            if (transaction.equalsIgnoreCase("1")) {
+                System.out.println("Available Balance is " + availableBal);
+            } else if (transaction.equalsIgnoreCase("2")) {
+                System.out.println("Select an amount to withdraw");
+                String withdraw = scanner.nextLine();
+                double withdrawAmount = Double.valueOf(withdraw);
+                availableBal = availableBal - withdrawAmount;
+                if (withdrawAmount > availableBal) {
+                    throw new Exception("Insufficient Funds.");
+                } else {
+                    System.out.println("Take your money. Your new balance is " + availableBal);
+                    //          }else if (transaction.equalsIgnoreCase("3"))
+                }
+            } else if (transaction.equalsIgnoreCase("3")) {
+                System.out.println("Thank you and please come again!");
+            }
```

Comment:
Jacob really well done.  You are one of the few students to get the looping right, which is one of the more difficult parts of this assignment. One minor thing the program should stop after the user enters the cancel option. You could do this in two ways.  One, throw an exception.  Two, exit the program.  The line of code that would do this is: System.exit(0); Again, its a minor detail great job.